demoapp/src/main/java/me/uport/sdk/demoapp/SignJWTUportHDSignerActivity.kt

```diff
@@ -0,0 +1,66 @@
+package me.uport.sdk.demoapp
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.uport.sdk.signer.UportHDSigner
+import com.uport.sdk.signer.UportHDSignerImpl
+import com.uport.sdk.signer.encryption.KeyProtection
+import com.uport.sdk.signer.importHDSeed
+import kotlinx.android.synthetic.main.create_import_key.*
+import kotlinx.coroutines.experimental.GlobalScope
+import kotlinx.coroutines.experimental.launch
+import me.uport.sdk.core.UI
+import me.uport.sdk.jwt.JWTTools
+
+class SignJWTUportHDSignerActivity : AppCompatActivity() {
+
+    var issuerDID: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.create_import_key)
```

Comment:
Since this layout is getting reused, it's worth renaming it to something more generic like:
`simple_result_layout` and also renaming the elements to not refer specifically to keys.

If that is not acceptable, please create a separate layout with proper elements to avoid confusion.